core/controllers/collection_editor.py

```diff
@@ -66,6 +66,7 @@ def get(self, collection_id):
             'SHOW_COLLECTION_NAVIGATION_TAB_STATS': (
                 feconf.SHOW_COLLECTION_NAVIGATION_TAB_STATS),
             'TAG_REGEX': feconf.TAG_REGEX,
+            'title': collection.title,
```

Comment:
@anubhavsinha98, I'm not sure whether this will fix the issue, as this handler is related to creating a collection and we are having an issue with updating the collection title of the existing collection. have you verified the changes with the "Steps to reproduce" given in the issue thread?

Reply:
I think you meant @abhaygupta97, @DubeySandeep :P.

Reply:
@abhaygupta97, I agree that it's working fine with refreshing the tab, but what if you just update the collection title without saving draft (i.e, without refresh), will that get reflected?

Reply:
@DubeySandeep if title is updated without refresh, the change does not get reflected. This feature is not present in Exploration editor also. I think this [issue](https://github.com/oppia/oppia/issues/1970) is related to changing collection title without refresh.

Reply:
@abhaygupta97, Thanks for checking!